discord_client.py

```diff
@@ -118,13 +117,13 @@ def run_loop(self):
             if file:
                 discord_message = await self.send_file(config.ch[msg.chat.id], file[0], filename=file[1], content=msg.content)
             else:
-                discord_message = await self.send_message(config.ch[msg.chat.id], msg.content)
+                discord_message = await config.ch[msg.chat.id].send(msg.content)
             self.update_internal_msg(msg, discord_message)
         except KeyError:
             logging.warning("Deleted a message from unkown chat.")
         except Exception as e:
             logging.exception("Exception while sending discord message")
-            self.forward_q.append((msg, file, work))
+            #self.forward_q.append((msg, file, work))
```

Comment:
Why commented out? 
The main reason for the code was to ensure that the bot will never drop any message, when discord has a rough time with server up time (like many times in the past)

Reply:
In case of programming errors in the bot, it would reschedule the message over and over again and never succeed in sending it, using 100% CPU on my system. At least a little delay would be nice, then.